-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-37262/29064

ERROR: Build errors found during clang-tidy run.
```
RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc:263:21: error: comparison of integers of different signs: 'int' and 'size_t' (aka 'unsigned long') [clang-diagnostic-sign-compare]
  for (iPF = 0; iPF < n; ++iPF) {
                    ^
Suppressed 1355 warnings (1351 in non-user code, 3 NOLINT, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc
+++ b/RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc
@@ -94,7 +94,7 @@ void HiBadParticleCleaner::produce(edm::StreamID, edm::Event& iEvent, const edm:
 
   size_t n = pfCandidates->size();
   std::vector<int> oldToNew(n);
-  int iPF = -1;
+  size_t iPF = -1;
   for (const reco::PFCandidate& pfCandidate : *pfCandidates) {
     iPF++;
     if (pfCandidate.particleId() == reco::PFCandidate::ParticleType::mu)  // muon cleaning

